main.cpp

```diff
@@ -52,85 +52,98 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
+//  Noun: vending machine
+//  action 1: dispense soda
+vendingMachine.dispenseSoda();
 
-//  action 2:
 
-//  action 3:
- 
+//  action 2: dispense water
+vendingMachine.dispenseWater();
+
+//  action 3: sound alarm
+ vendingMachine.soundAlarm();
+
 //  2)
-//  Noun:
-//  action 1:
+//  Noun: Rock Paper Scissors Player
+//  action 1: player plays rock
+player.playRock();
+//  action 2: player plays scissors
+player.playScissors();
 
-//  action 2:
+//  action 3: player plays paper
+player.playPaper(); 
 
-//  action 3:
- 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: kickBoxer
+//  action 1: kick
+kickBoxer.kick();
+//  action 2: box
+kickBoxer.box();
+//  action 3: hadouken
+kickBoxer.hadouken();
 
-//  action 3:
- 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
-//  6)
-//  Noun:
-//  action 1:
+//  Noun: student
+//  action 1: complete assignment
+student.completeAssignment();
+//  action 2: sleep
+student.sleep();
+//  action 3: procrastinate
+student.procrastinate();
+
+//  5) 
+//  Noun: cow
+//  action 1:  eat grass
+cow.eatGrass();
+//  action 2: eat grain
+cow.eatHay();
+//  action 3: die
+cow.die();
```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**REPLACE** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.
**Do not** create an 11th noun.